Makefile

```diff
@@ -2,8 +2,12 @@
 
 build-dependencies:
 	go get github.com/beego/bee/v2
+	go get github.com/wellington/wellington/wt
 	go mod tidy
 
+build-assets:
+	wt compile assets/stylesheets/application.scss -s compressed -b static/css
```

Comment:
Do you need to execute this command each time? No live reloading in place?

Reply:
It has `watch` command also. Updated with it in here https://github.com/junan/google-scraper/pull/25/commits/72877b0ad43612ec659544f7bab95df7d6237d8b